packages/configuration-validation/babel.config.js

```diff
@@ -0,0 +1,10 @@
+const presets = ['@babel/preset-env'];
+const plugins = [];
+
+// Process typescript when running in jest
+if (process.env.NODE_ENV === 'test') {
```

Comment:
Does the `root` option work here? The test config can be moved into /test folder.

Reply:
The tests seem to work fine as .ts files using `babel-jest`. I'm not sure about moving the babel config. I copied this from okta-auth-js. I think the `root` option would apply if we were running the tests from the `/tests` directory (I think it  goes with **working** directory, not **src** directory